modules/browser-node-test-ui/config-node.json

```diff
@@ -2,7 +2,6 @@
   "adminToken": "cxt1234",
   "authUrl": "http://localhost:5040",
   "logLevel": "info",
-  "mnemonic": "dial accuse that borrow dinosaur arrange erode dilemma want citizen nuclear hurt",
```

Comment:
dont you still need a mnemonic included in the config?

Reply:
Nope, it's part of the UI now.